dialogue/TownNPCs/ShovelGuy/ShovelGuy2.json

```diff
@@ -21,7 +21,7 @@
     ],
     "pl": [
       "Już pracowałem nad tym przez jakiś czas, i nie wygląda na to, że skończę w najbliższym czasie.",
```

Comment:
"Już pracuję nad tym jakiś czas". Also I'd change "że skończę w najbliższym czasie" to "że szybko skończę" to don't repeat the word "czas" too much